synapse/handlers/presence.py

```diff
@@ -744,7 +750,9 @@ async def set_state(self, target_user, state, ignore_status_msg=False):
             msg = status_msg if presence != PresenceState.OFFLINE else None
             new_fields["status_msg"] = msg
 
-        if presence == PresenceState.ONLINE:
+        if presence == PresenceState.ONLINE or (
+            self._busy_presence_enabled and presence == PresenceState.BUSY
```

Comment:
Maybe for consistency?

```suggestion
            presence == PresenceState.BUSY and self._busy_presence_enabled
```